src/Language/PureScript/Publish/ErrorsWarnings.hs

```diff
@@ -54,11 +54,13 @@ data PackageWarning
 -- | An error that should be fixed by the user.
 data UserError
   = BowerJSONNotFound
+  | LicenseNotFound
   | BowerExecutableNotFound [String] -- list of executable names tried
   | CouldntDecodeBowerJSON (ParseError BowerError)
   | TagMustBeCheckedOut
   | AmbiguousVersions [Version] -- Invariant: should contain at least two elements
   | BadRepositoryField RepositoryFieldError
+  | MissingLicenseField
```

Comment:
Bit of a nit, but I'd rather call this constructor `NoLicenseSpecified`. Suppose someone puts `license: []` in their bower.json. In this case, there would be a license *field*, but this error would (correctly) be thrown, as no license would have been specified.